src/ptr/CowSlice.zig

```diff
@@ -0,0 +1,270 @@
+const std = @import("std");
```

Comment:
for future code, remember that files should be named snake_case unless they are a top level struct. this makes it easier to understand what `@import` returns without having to refer to the source contents